web3j-ext/web3j-ext/src/main/java/org/web3j/example/FeeDelegatedCancelExample.java

```diff
@@ -59,20 +60,18 @@ public static void run() throws IOException {
         EthSendTransaction transactionResponse = web3j.ethSendRawTransaction(hexValue).send();
         System.out.println("TxHash : \n " + transactionResponse.getResult());
         String txHash = transactionResponse.getResult();
-        try
-        {
-             Thread.sleep(2000);
-        }
-        catch(Exception e)
-        {
+        try {
```

Comment:
Is it possible to replace it to polling mechanism by receipt manager?

Reply:
i will change all examples with pollingTransactionReceiptProcessor on next PR